src/test/kotlin/com/jetbrains/snakecharm/lang/parser/SnakemakeParsingTest.kt

```diff
@@ -221,6 +221,7 @@ class SnakemakeParsingTest : ParsingTestCase(
         doTest()
     }
 
+
```

Comment:
Please review diff before commit, this empty line was occasionally introduced in code and committed with other changes